workers/analyzer/src/test/kotlin/AnalyzerRunnerTest.kt

```diff
@@ -565,8 +565,8 @@ class AnalyzerRunnerTest : WordSpec({
             val secret2 = mockk<Secret>()
             val environmentConfig = ResolvedEnvironmentConfig(
                 environmentVariables = setOf(
-                    EnvironmentVariableDefinition("MY_ENV_VAR", secret1),
-                    EnvironmentVariableDefinition("ANOTHER_ENV_VAR", secret2)
+                    SecretVariableDefinition("MY_ENV_VAR", secret1),
+                    SecretVariableDefinition("ANOTHER_ENV_VAR", secret2)
```

Comment:
There should be a test case as well for a `SimpleVariableDefinition`.

Reply:
I have added SimpleVariableDefinition to the already existing Text "create a process builder with the correct environment". Because imo it would just duplicate unnecessary code to create a new test. Is that fine?